### DISCUSSION TITLE
Password recovery/activation email

### DISCUSSION THREAD

**Discussion by EmmaHudee** [globaleaks/globaleaks-whistleblowing-software#4545] [General]
Hello,

I am experiencing various issues when trying to set up GlobaLeaks. The installation by itself got successefully completed. However, the following issues remain:

First, I have not received any emails when the acount was created on my adminitrator account. Therfore, I do not have the key required to recover forgotten passwords.

Second, my users cannot have access to the website when trying to use their activation link from their emails.

Third, I have deconnected from my account (as an administrator) and despite the correct password and the correct email address, I cannot reconnect. 

For the first issue, I have tried to do everything both through http access and tor. However, for the second and third problems, I cannot access the login page within the tor framework because it has never been sent or generated.

In advance, thank you for your help.

P.S.: I have also tried to go through the contact access to request support but the page isn't available.

**Comment by evilaliv3** [comment]
Thank you @EmmaHudee for reaching us here.

This is the correct place where to open software issues (if any); For community support you may use the community chat: https://slack.globaleaks.org

As for your issues:
- First question answer: you can log in as administrator you can go in your preferences page and get your account recovery key clicking the button: "Account recovery key"
- Second queston answer: would you provide the link and describe the behaviour that your users have when clicking it?
- Thirt question: actually i htink you may have wrongly typed the password when setting/changing it and you may be currently locked out. Did you create other admin users?

If the system is live but with no submissions on it we may be able to reset the situation and grant you access again removing existing encryption keys. Eventually please reach us on slack.

thank you

**Comment by EmmaHudee** [comment]
Thank you for your answer. In order to clarify some aspects of my questions:

First, In order to have access to the key, I need to have access to the account but since I don't have the password (the reason for which I need the key in the first place) I cannot download it back.

Second, talking about the link for the activation of the account, nothing happen, the page does not load.

Third, Only one admin account was created, I am sure of the password as I had it saved when creating the account to avoid such mistake when typing it. Moreover, after a few hours of the tor link to access the website to drop off the document worked fine, it just stopped working, not accessing the page anymore as if the page never existed.

Thank you for your help.

**Comment by evilaliv3** [comment]
Thank you @EmmaHudee 

I suggest you to setup again the platform from scratch and try to store the recovery key this time as first thing.
I'm working on a patch to guide the user performing this at their first login.

I'm closing this ticket as it is now full of different topics. Please feel free to open new tickets, one for each topic.

Thank you!
